pkg/pillar/cmd/volumemgr/create.go

```diff
@@ -76,17 +74,7 @@ func createVdiskVolume(ctx *volumemgrContext, status types.VolumeStatus,
 		}
 	}()
 
-	if persistFsType != types.PersistZFS {
-		if _, err := os.Stat(filelocation); err == nil {
-			errStr := fmt.Sprintf("Can not create %s for %s: exists",
-				filelocation, status.Key())
-			log.Error(errStr)
-			return created, "", errors.New(errStr)
-		}
```

Comment:
Without this check will we catch the case of an existing volume elsewhere, or will we recreate the volume even if it exists?

Reply:
I moved this check into if-else branch few lines below, so we will have the same behavior.